long-term-storage.md

```diff
@@ -1,83 +1,111 @@
-Long Term Storage
-=================
+Multi-Cluster / Long Term Storage
+=================================
 
-To enable 90+ days of data retention in Kubecost, we recommend deploying with durable storage enabled. We provide two options for doing this: 1) in your cluster and 2) out of the cluster. This functionality also powers the Enterprise multi-cluster view, where data across clusters can be viewed in aggregate, as well as simple backup & restore capabilities.
+Kubecost leverages Thanos to enable durable storage for three different purposes:
 
-> Note: This feature requires an [Enterprise license](https://kubecost.com/pricing).
-
-Out of cluster storage (Thanos)
+1. Centralize metric data for a global multi-cluster view into Kubernetes costs via a Prometheus sidecar
+1. Allow for unlimited data retention
+1. Backup Kubecost [ETL data](https://guide.kubecost.com/hc/en-us/articles/4407601815191-ETL-S3-Backup)
 
-Thanos-based durable storage provides long-term metric retention directly in a user-controlled bucket (e.g. S3 or GCS bucket) and can be enabled with the following steps:
+> Note: This feature requires an [Enterprise license](https://kubecost.com/pricing).
 
-### Step 1: Create object-store yaml file
+To enable Thanos, follow these steps:
+### Step 1: <a name="storage"></a>Create object-store.yaml
 
-This step creates the object-store.yaml file that contains your durable storage target (e.g. GCS, S3, etc.) configuration and access credentials. 
+This step creates the object-store.yaml file that contains your durable storage target (e.g. GCS, S3, etc.) configuration and access credentials.
 The details of this file are documented thoroughly in [Thanos documentation](https://thanos.io/tip/thanos/storage.md/).
 
+We have guides for using cloud-native storage for the largest cloud providers. Other providers can be similarly configured.
+
 Use the appropriate guide for your cloud provider:
 * [Google Cloud Storage](https://github.com/kubecost/docs/blob/main/long-term-storage-gcp.md)
 * [AWS/S3](https://github.com/kubecost/docs/blob/main/long-term-storage-aws.md)
 * [Azure](https://github.com/kubecost/docs/blob/main/long-term-storage-azure.md)
 
-### Step 2: Create object-store secret
+### <a name="secret"></a>Step 2: Create object-store secret
 
-The final step prior to installation is to create a secret with the yaml file generated in the previous step:
+Create a secret with the yaml file generated in the previous step:
 
 ```shell
 kubectl create secret generic kubecost-thanos -n kubecost --from-file=./object-store.yaml
 ```
 
-### Step 3: Deploying Kubecost with Thanos
+### <a name="cluster_id"></a>Step 3: Unique Cluster ID
+
+Each cluster needs to be labelled with a unique Cluster ID, this is done in two places.
+
+`values-clusterName.yaml`
+```yaml
+kubecostProductConfigs:
+  projectID: yourClusterName
```

Comment:
If we are standardizing on `kubecostProductConfigs_clusterName` it should be used here.